README.md

```diff
@@ -5,8 +5,10 @@
 ## Introduction
 
 This is the implementation of
-the [Electronic Health Certificates (EHN) specification](https://github.com/ehn-digital-green-development/hcert-spec)
-used to verify the validity of COVID Certificates [in Switzerland](https://github.com/admin-ch/CovidCertificate-App-Android).
+the [Electronic Health Certificates (EHN)](https://github.com/ehn-digital-green-development/hcert-spec)
```

Comment:
`Electronic Health Certificates (EHN)`
should be 
`Electronic Health Certificates (HCERT)`.
The EHN is the EU's "eHealth Network". We should change this in the other repos as well at some point.